src/write-docs-from-tests.js

```diff
@@ -40,6 +41,7 @@ const { globalTemplates, loadRuleTemplate } = loadTemplates( templatePaths );
 function writeDocsFromTests( name, rule, tests, testerConfig, done ) {
 	const configMap = rulesWithConfig.get( name ).configMap;
 	const outputPath = packagePath( config.docPath.replace( '{name}', name ) );
+	const outputDir = packagePath( path.dirname( config.docPath ) );
```

Comment:
Actually `{name}` can be part of a folder name, so this should the the dirname of the replace'd outputPath:
`path.dirname(outputPath)`

Fixed in #93